contrib/charts/cert-manager/templates/hook_apply_issuers.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.clusterIssuer.email }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: apply-issuers
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: apply-issuers
+      namespace: {{ .Release.Namespace }}
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: apply-issuers
+        image: claranet/gcloud-kubectl-docker
+        imagePullPolicy: Always
+        command: ["kubectl","apply","-f","/tmp/"]
+        volumeMounts:
+          - name: issuers
+            mountPath: /tmp
```

Comment:
`/tmp` is a bad path since it's already overloaded to have other semantics.

An example of something that could break here: if `kubectl` internally decided to use `mktemp`, it might create files here at runtime which it would then see.

I'd much rather this be a path like `/manifests` or pretty much any other non-special path.

Reply:
I used /issuers